lib/execute-method-map.ts

```diff
@@ -333,6 +333,24 @@ export const executeMethodMap = {
       required: ['style'],
     },
   },
+  'mobile: increaseContrast': {
```

Comment:
I find this method name confusing since it contains a verb. We need to either keep `get` prefix or have `is` prefix to its name